web/biome.json

```diff
@@ -10,6 +10,7 @@
       "**/tsconfig*",
       "libs/editor/examples/**/*.json",
       "libs/editor/src/examples/**/annotations/1.json",
+      "libs/ui/src/shad/**/*",
```

Comment:
why do we ignore these new components? they will essentially be a part of our application, the code we take full responsibility of, so let's make it up to our standards.

Reply:
makes sense